pyproject.toml

```
@@ -49,6 +49,7 @@ package-dir = {"" = "src"}
 
 [tool.setuptools.dynamic]
 dependencies = {file = ["requirements.txt"]}
+optional-dependencies.hpu = { file = ["requirements-hpu.txt"]}
```

```suggestion
optional-dependencies.hpu = { file = ["requirements-hpu.txt"] }
```
Why do we add this now?

---

We are adding this again, so `pip install instructlab[hpu]` is possible again. We had to remove the optional dependencies because we previously required a git version of optimium-habana. PyPI does not allow packages with direct URL references.